### PR TITLE
Revert gen onnx mlir script

### PR DESCRIPTION
**Pull request by doru1004**
Fix gen onnx mlir script which was messed up by a previous auto format.

### CODE REVIEW

File: utils/gen_onnx_mlir.py [hunk 1]
```diff
@@ -35,229 +35,228 @@
...
+
 # check the version of onnx package being used
-if current_onnx_version != onnx.__version__:
-    print("version of expected onnx is {}, ".format(current_onnx_version) +
+if current_onnx_version != onnx.__version__ :
+    print("version of expected onnx is {}, ".format(current_onnx_version)+
           "while onnx package being used is {}".format(onnx.__version__))
     quit()
 
 current_onnx_version = "1.9.0"
```
Comment (sstamenova) [inline utils/gen_onnx_mlir.py:53]: This version string is in 3 separate places, is that normal? It seems wrong.

File: utils/gen_onnx_mlir.py [hunk 2]
```diff
@@ -35,229 +35,228 @@
...
 # check the version of onnx package being used
-if current_onnx_version != onnx.__version__:
-    print("version of expected onnx is {}, ".format(current_onnx_version) +
+if current_onnx_version != onnx.__version__ :
```
Comment (sstamenova) [inline utils/gen_onnx_mlir.py:55]: This check is also here 2? 3? times. This seems like another merge issue.

File: utils/gen_onnx_mlir.py [hunk 3]
```diff
@@ -809,48 +805,44 @@ def get_type_inference_func(s, indent, type_inference_code):
...
+        'uint64' : 'UI64',
+        'int8' : 'I8',
+        'int16' : 'I16',
+        'int32' : 'I32',
+        'int64' : 'I64',
+        'float16' : 'F16',
+        'bfloat16' : 'BF16',
+        'float' : 'F32',
+        'double' : 'F64',
+        'unkown' : 'BF16',
```
Comment (sstamenova) [inline utils/gen_onnx_mlir.py:833]: Nit: unknown

**Comment by doru1004** [comment]
@sstamenova I am not changing the script in any way, just reverting it since my previous commit accidentally changed the formatting. The fixing of the problems in the script should be done in a separate patch.

**Review by AlexandreEichenberger — APPROVED** [review]
LGTM, I don't mind if you want to revert the formatting. However, I would not mind to clangFormat python scripts too, even if originally due to an accident.

**Comment by sstamenova** [comment]
> @sstamenova I am not changing the script in any way, just reverting it since my previous commit accidentally changed the formatting. The fixing of the problems in the script should be done in a separate patch.

Thanks for changing the title, it's clearer now.

**Comment by doru1004** [comment]
@jenkins-droid test this please

**Pull request merged**